tests/fiaas_deploy_daemon/deployer/kubernetes/test_ready_check.py

```diff
@@ -232,10 +265,141 @@ def test_tls_ingress_not_ready(self, get, app_spec, bookkeeper, lifecycle, lifec
                 lifecycle.success.assert_not_called()
 
     def test_deployment_tls_config_no_tls_extension(self, get, app_spec, bookkeeper, lifecycle,
-                                                    lifecycle_subject, config):
+                                                    lifecycle_subject, ingress_adapter, config):
+        config.tls_certificate_ready = True
+        self._create_response(get)
+        ready = ReadyCheck(app_spec, bookkeeper, lifecycle, lifecycle_subject, ingress_adapter, config)
+
+        assert ready() is False
+        bookkeeper.success.assert_called_with(app_spec)
+        bookkeeper.failed.assert_not_called()
+        lifecycle.success.assert_called_with(lifecycle_subject)
+        lifecycle.failed.assert_not_called()
+
+    def test_tls_ingress_ready_v1(self, get, app_spec, bookkeeper, lifecycle, lifecycle_subject, ingress_v1_adapter,
```

Comment:
It would be nicer to parametrize the tests so you can test both versions with the same method, and avoid boilerplate